CC/src/DistanceComputationTools.cpp

```diff
@@ -2328,7 +2450,7 @@ int DistanceComputationTools::computeCloud2ConeEquation(GenericIndexedCloudPersi
 	{
 		*rms = sqrt(dSumSq / count);
 	}
-	return count;
+	return DISTANCE_COMPUTATION_RESULTS::SUCCESS;
```

Comment:
Was this intended? Is anything expecting this to return a count?

This change occurs several times below.

Reply:
No, I wrote the primitive distance functions originally and just intended to give a positive success, these are only used in the `doActionCloudPrimitiveDist` function in mainwindow.

Reply:
by No I mean: No nothing is expecting this to return a count.
Yes this was intentional.